File: java/lib/src/main/java/com/svix/internalapi/Destination.java
// this file is @generated
package com.svix.internalapi;

import com.svix.SvixHttpClient;

import lombok.Getter;

public class Destination {
    private final SvixHttpClient client;

    @Getter private final DestinationAutoconfig autoconfig;

    public Destination(SvixHttpClient client) {
        this.client = client;
        this.autoconfig = new DestinationAutoconfig(client);
    }
}

File: php/src/ApiInternal/Destination.php
<?php

// this file is @generated
declare(strict_types=1);

namespace Svix\ApiInternal;

use Svix\Request\SvixHttpClient;

class Destination
{
    public DestinationAutoconfig $autoconfig;

    public function __construct(
        private readonly SvixHttpClient $client,
    ) {
        $this->autoconfig = new DestinationAutoconfig($client);
    }
}
